app/src/main/index.js

```diff
@@ -197,9 +197,35 @@ function resetTrayIcon() {
   mainWindow.setAlwaysOnTop(false);
 }
 
+const animateIcon = () => new Promise(resolve => {
+  const interval = 40;
+  let i = 0;
+
+  const next = () => {
+    setTimeout(() => {
+      const number = String(i++).padStart(5, '0');
```

Comment:
I think something like `frameNumber` would be more descriptive 👍

Reply:
Maybe better to have a function that generates the filename from the frame number + animation name?